### DISCUSSION TITLE
jobrunr - distributed processing

### DISCUSSION THREAD

**Discussion by chaudharydeepak** [jobrunr/jobrunr#810] [Q&A]
Hi, I recently started looking into jobrunr and was very impressed with easiness / capabilities of the library(?).

is it possible to use jobrunr for following scenario -

1. App1 receives some events, and we want to process these events. The responsibility to process these events is on App2. 
2. So, App1 receives events, create jobs out of them, submit(persist) the job and thats it. 
3. The App2 is listening for new jobs created - and will execute if there are any. The job execution logic is part of the task itself.

So, with jobrunr, is it possible to create background tasks in app1 and have them processed by app2 - ofcourse both app1 and app2 shares same persistance ( db )? Thank you for your help.

**Comment by rdehuyss** [comment]
Yes, see the Spring Boot example. https://github.com/jobrunr/example-spring

**Comment by chaudharydeepak** [comment]
Thank you @rdehuyss . will check it out.
